spec/models/course_spec.rb

```diff
@@ -78,7 +78,111 @@
     end
   end
 
-  describe '#get_assignment_list' # TODO
+  def create_assignment_csv_string(a)
```

Comment:
It's good to make helper functions here, but please move them to the bottom of this file and make them `private` to this test class (https://www.rubyguides.com/2018/10/method-visibility/).

In general do not use single-letter parameter names. Prefer the model name when possible (so `assignment` instead of `a`).